ui/web-v2/apps/admin/src/lang/messages.ts

```diff
@@ -504,6 +504,106 @@ export const messages = {
       defaultMessage: 'Evaluation Count',
     }),
   },
+  trigger: {
+    documentation: defineMessage({
+      id: 'trigger.documentation',
+      defaultMessage: 'documentation',
+    }),
+    description: defineMessage({
+      id: 'trigger.description',
+      defaultMessage:
+        'Use triggers to turn a flag on or off remotely. See the {link}',
```

Comment:
Please update the documentation link.
https://docs.bucketeer.io/feature-flags/creating-feature-flags/triggers